clippy_lints/src/lib.rs

```diff
@@ -303,6 +303,7 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry<'_>, conf: &Conf) {
     reg.register_late_lint_pass(box block_in_if_condition::BlockInIfCondition);
     reg.register_late_lint_pass(box unicode::Unicode);
     reg.register_late_lint_pass(box strings::StringAdd);
+    reg.register_late_lint_pass(box strings::StringRepeatOnce);
```

Comment:
You also have to add the Lint to the right point group. Running the `util/update_lints.py` script should do this for you!